pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -374,6 +374,9 @@ func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
 	if obj.EvictionPressureTransitionPeriod == zeroDuration {
 		obj.EvictionPressureTransitionPeriod = metav1.Duration{Duration: 5 * time.Minute}
 	}
+	if obj.ExperimentalKernelMemcgNotification == nil {
```

Comment:
yeah, i just fixed a push with that.  i was still messing w/ generators and just threw this up to make people aware i was working on it.